kong/runloop/cache_prewarm.lua

```diff
@@ -52,6 +64,14 @@ function cache_prewarm.execute()
   for _, entity_name in ipairs(ENTITIES_TO_PREWARM) do
     local ok, err = cache_prewarm_single_entity(entity_name)
     if not ok then
+      if err == "no memory" then
+        kong.log.warn("cache memory is exhausted, please consider increasing ",
```

Comment:
It would have been helpful to explain that this happened during cache warming, and maybe explain what the implications of this are. It's important users don't get mislead into believing that a full cache could mean that Kong would fail to serve requests or any such thing...